packages/hash/external-services/kratos/Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM oryd/kratos:v0.10.1
```

Comment:
Guessing you will update this as part of [this PR](https://github.com/hashintel/hash/pull/1635/files) if this one merges first?

Reply:
Yes, I already have a rebased version ready in case this got in first :)